### PR TITLE
Update nexusmods.com.xml (fixes #11953)

### LINKED ISSUE #11953: nexusmods.com rule breaks downloading
It seems commit 27fef948c9fd3ed653581b1fa77ff14697b309d7 is breaking downloading on nexusmods.com (on Chrome) since some issues with CORS and Mixed Content.

```
Uncaught DOMException: Failed to execute 'pushState' on 'History': A history state object with URL 'http://www.nexusmods.com/fallout4/mods/24491/?tab=2&navtag=http%3A%2F%2Fwww.nexusmods.com%2Ffallout4%2Fajax%2Fmodfiles%2F%3Fid%3D24491&pUp=1' cannot be created in a document with origin 'https://www.nexusmods.com' and URL 'https://www.nexusmods.com/fallout4/mods/24491'.

Mixed Content: The page at 'https://www.nexusmods.com/fallout4/mods/24491' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://www.nexusmods.com/fallout4/ajax/downloadfile?id=106843'. This content should also be served over HTTPS.

Failed to load https://www.nexusmods.com/fallout4/ajax/downloadfile?id=106843: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'null' is therefore not allowed access.
```

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by cschanaj** [comment]
also fix #11970

**Comment by koops76** [comment]
@cschanaj Likely the same issue.

**Comment by umisery** [comment]
How do I apply this fix?

**Comment by koops76** [comment]
@umisery Simply wait until it will be merged, the next release will have it. Until then disable NexusMods.com ruleset.

**Comment by umisery** [comment]
Alright, thanks.

**Comment by springacres** [comment]
@umisery also, make sure to disable the ruleset before you log in to any nexusmods.com site.

**Comment by umisery** [comment]
I have no idea how to do either of those, so I'm just going to wait for it to be merged :P

File: src/chrome/content/rules/nexusmods.com.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+		- ^
+		- www
+		- wiki
 -->
 
 <ruleset name="NexusMods.com">
-	<target host="nexusmods.com" />
-	<target host="www.nexusmods.com" />
 	<target host="filedelivery.nexusmods.com" />
 	<target host="forums.nexusmods.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/nexusmods.com.xml]: Can we add `staticdelivery.nexusmods.com`?

File: src/chrome/content/rules/nexusmods.com.xml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <!--
 	Mixed content:
```
Comment (J0WI) [inline src/chrome/content/rules/nexusmods.com.xml]: Is it worth to create a MCB platform rule for them?

**Comment by J0WI** [comment]
To fix the mentioned issues I think we can also add an exclusion. Other parts of the site WFM.

**Comment by koops76** [comment]
@Hainish Merge before the release, NexusMods.com is broken.

**Comment by Hainish** [comment]
We should set a policy on what rules are worthy of blocking release for.  Looks like this one ranks 1,070, hardly a show-blocker.

**Comment by J0WI** [comment]
This should not be a blocker for the current release. #12268 has a much bigger impact than any ruleset.

@koops76 I already commented on your changes. When we remove the entry host we don't fix anything, we just disable it.

**Comment by cschanaj** [comment]
In https://github.com/EFForg/https-everywhere/issues/11953#issuecomment-326191685, it is mentioned that 
> To confirm, Nexus Mods does not currently support HTTPS on the main site as per the following forum thread: https://forums.nexusmods.com/index.php?/topic/5907893-site-problems-an-error-occurred/

A number of users have experienced issues with this site (see #11953, #12324 and #12505). IMHO, We should disable this ruleset if it is impossible to fix. (Possibly with `default_off="other"` and a reference to the forum link). thanks.

File: src/chrome/content/rules/nexusmods.com.xml (hunk 2 above)
Comment (koops76) [reply src/chrome/content/rules/nexusmods.com.xml]: Maybe but unlikely. I don't think anyone actually has active mixed content blocking disabled after Tor Browser enabled it. I think `platform="mixedcontent"` rulesets are on their way out.

**Pull request closed without merging**